static/js/apps/nl_interface/query_result.tsx

```diff
@@ -149,7 +149,9 @@ export const QueryResult = memo(function QueryResult(
           </a>
           {chartsData && chartsData.sessionId && (
             <span
-              className={`thumb-down ${isThumbClicked ? "thumb-dim" : ""}`}
+              className={`feedback-emoji ${
+                isThumbClicked ? "feedback-emoji-dim" : ""
```

Comment:
Is the intended behavior for query_result to be specific to thumbs down?

If not, let's use `isEmojiClicked` instead and update the function on line 200 to onChartClick to match nl_feedback.tsx

Reply:
Yes, thumbs-down for now.